com-dict-client/.env

```diff
@@ -1 +1,4 @@
-
+REACT_APP_FIREBASE_API_KEY = "AIzaSyBcd0O5iQ1EjtJK9bBg-zLeC_HBukeVfSA"
```

Comment:
You've already committed the keys here. Now there are three things to do.

1. Delete this commit and recommit ignoring the .env file.
2. Add rules in firebase so that showing the credentials won't affect the security of the firebase project.
3. Delete the firebase project and create a new one with a new name.